code/modules/urist/modules/shipbattles/overmapships/hostile_ships.dm

```diff
@@ -0,0 +1,160 @@
+/mob/living/simple_animal/hostile/overmapship/pirate
+	wander = 1 //temporary
+	color = "#bb0000"
+	hiddenfaction = /datum/factions/pirate
+	aggressive = 1
+	boarders_amount = 3
+
+/mob/living/simple_animal/hostile/overmapship/pirate/small
+//	shipdatum = /datum/ships/piratesmall
+	shields = 800
```

Comment:
```suggestion
	shields = 750
```
Mismatch with `/datum/shipcomponents/shield/light` causes the shipcombat UI to incorrectly display 750/800.

Reply:
I should investigate this more closely before I go commit your suggestion.

Reply:
Relevant code is https://github.com/Haven-13/Haven-Urist/blob/de25ecd28982470bda658a59ab43196f7024682a/code/modules/urist/modules/shipbattles/combatcomputer.dm#L85
Ideally it'd just be acquired from the shield's var instead of having the ship's default always match up.